### PR TITLE
Fix parachain-template-test

### PR DESCRIPTION
**Pull request by pepoviola**
Fix `parachain-template-test` (bump `zombienet` version).
Thx!

### CODE REVIEW

File: templates/zombienet/Cargo.toml [hunk 1]
```diff
@@ -11,10 +11,10 @@ publish = false
...
 [dependencies]
 env_logger = "0.11.2"
-log = "0.4"
-tokio = { version = "1.36.0", features = ["rt-multi-thread"] }
-anyhow = "1.0.81"
-zombienet-sdk = "0.2.8"
+log = { workspace = true }
+tokio = { workspace = true, features = ["rt-multi-thread"] }
+anyhow = { workspace = true }
+zombienet-sdk = "0.2.13"
```
Comment (alexggh) [inline templates/zombienet/Cargo.toml:17]: Shouldn't we put this one in the workspace as well ?

**Review by alexggh — APPROVED** [review]

File: templates/zombienet/Cargo.toml (hunk 1 above)
Comment (pepoviola) [reply templates/zombienet/Cargo.toml:17]: Yes, I will move it 👍 .

File: polkadot/zombienet-sdk-tests/Cargo.toml [hunk 2]
```diff
@@ -9,15 +9,15 @@ publish = false
 
 [dependencies]
 env_logger = "0.11.2"
-log = "0.4"
+log = { workspace = true }
 subxt = { version = "0.37", features = ["substrate-compat"] }
 subxt-signer = { version = "0.37" }
```
Comment (bkchr) [inline polkadot/zombienet-sdk-tests/Cargo.toml:14]: All deps should be workspace deps. No dep should be crate local anymore.

Comment (pepoviola) [reply polkadot/zombienet-sdk-tests/Cargo.toml:14]: Sounds good, thanks for the feedback 👍

**Review by bkchr — APPROVED** [review]

**Pull request merged**